Blog post of Inaugural OPA winners (#664)
pyladies/pyladies: www/_posts/2024/2024-05-28-outstanding-pylady-winners.md

---
layout: post.html
title: "Inaugural Winners of the Outstanding PyLady Award"
tags: [PyLadies, Global, Community, Outreach]
author: Lynn Root
author_link: https://mastodon.online/@roguelynn
---

<TODO: photo of award winners at PyCon>

> I am sure many of us know and have experienced ourselves: the PyLadies community is made up of many
> remarkable members and volunteers that make significant contributions to bettering the Python
> community.

Reviewer: ```suggestion
The PyLadies community is made up of many
remarkable members and volunteers who make significant contributions to bettering the Python
community.
```

The intro felt a bit out of place

> To celebrate that, [Cheuk Ting Ho] proposed an idea to the PyLadies Global Council last year to 
> recognize remarkable PyLadies. Cheuk saw many folks are doing amazing things for the community
> and proposed the [Outstanding PyLady Award][blog announcement]. This new Outstanding PyLady Award
> is meant to elevate diverse Pythonistas doing extraordinary community-elevating things, to highlight
> their achievements, and to thank them for all the work they've contributed.
> 

Reviewer: ```suggestion
To celebrate that, [Cheuk Ting Ho] saw many folks doing amazing things for the community and proposed establishing the [Outstanding PyLady Award][blog announcement] to the PyLadies Global Council. This new Outstanding PyLady Award
is meant to elevate diverse Pythonistas doing extraordinary community-elevating things, highlight
their achievements, and to thank them for all the work they've contributed.

```

> Each year, the PyLadies Global Council will be selecting a handful of folks from nominations from

Reviewer: ```suggestion
Moving forward, every year the PyLadies Global Council will be selecting a handful of folks from nominations from
```

the Python community at large (anyone in the Python community is encouraged to 
[nominate][nomination form]!). Award winners are recognized at PyCon US, and receive their choice of
either free registration & travel assistance to PyCon US, or a cash award of $1,000. Its intention
is to demonstrate that their service and contributions to the PyLadies community does lead to
recognition and reward. They also get a cute little pin to wear on their lanyards for all PyCons to
come, if they wish. <TODO: image of pin>

> This is the first year of the Outstanding PyLady awards, and we got a whopping _**84 submissions**_ 
> for nominations. Unfortunately, we do not have the capacity to award all the nominations, although 
> all of them are very clearly deserving. 

Reviewer: ```suggestion
This is the first year of the Outstanding PyLady awards, and we got a whopping _**84 submissions**_ 
for nominations. Unfortunately, we do not have the capacity to award all the nominees, although 
all of them clearly deserve it. 
```

---
layout: post.html
title: "Inaugural Winners of the Outstanding PyLady Award"
tags: [Global, "Outstanding PyLady Award"]
author: Lynn Root
author_link: https://mastodon.online/@roguelynn
---

Announcing the first recipients of the Outstanding PyLady Awards (🎉 OPA 🎉)!

<figure>
    <img src="{{ get_asset('images/blog/opa_winners_2024.png') }}" alt="Award winners posing at PyCon US 2024 Closing Ceremony" width="70%" style="margin-left: auto; margin-right: auto; margin-bottom: 2em; margin-top: 2em; display: block;">

> <figcaption style="text-align: center;"><i>Abigail Mesrenyame Dogbe (left-most), Maaya Ishida (second from right), and Jessica Green (right-most) posing with their blue gem awards with Lynn Root (second from left) at the closing ceremony of PyCon 2024. Photo taken by <a href="https://x.com/psobot">@psobot</a>.</i></figcaption></figure>

Reviewer 2: ```suggestion
<figcaption style="text-align: center;"><i>Abigail Mesrenyame Dogbe (left-most), Maaya Ishida (second from right), and Jessica Greene (right-most) posing with their blue gem awards with Lynn Root (second from left) at the closing ceremony of PyCon 2024. Photo taken by <a href="https://x.com/psobot">@psobot</a>.</i></figcaption></figure>
```

I wrote this in the email but then realised I could just add it here :)
